go.mod

```diff
@@ -29,6 +29,29 @@ require (
 	sigs.k8s.io/controller-runtime v0.18.4
 )
 
+require (
```

Comment:
Nit -

There should only be two `require` blocks (for direct and indirect) but I'm guessing go is getting confused by the replace directive? I wonder if `go mod tidy` consolidates these after the `replace` is moved to the top or bottom of the file.

Reply:
Updated the `go.mod` file by manually deleting the `indirect` block and letting it regenerate.